ui/tests/acceptance/client-history-test.js

```diff
@@ -275,4 +274,30 @@ module('Acceptance | clients history tab', function (hooks) {
     assert.dom('[data-test-popup-menu-trigger="month"]').exists('Dropdown exists to select month');
     assert.dom('[data-test-popup-menu-trigger="year"]').exists('Dropdown exists to select year');
   });
+
+  test('shows error template if permissions denied querying activity response', async function (assert) {
+    this.server = new Pretender(function () {
+      this.get('/v1/sys/license/status', () => sendResponse(null, 403));
+      this.get('/v1/sys/version-history', () => sendResponse(null, 403));
+      this.get('/v1/sys/internal/counters/config', () => sendResponse(null, 403));
+      this.get('/v1/sys/internal/counters/activity', () => sendResponse(null, 403));
+      this.get('/v1/sys/health', this.passthrough);
+      this.get('/v1/sys/seal-status', this.passthrough);
+      this.post('/v1/sys/capabilities-self', this.passthrough);
+      this.get('/v1/sys/internal/ui/feature-flags', this.passthrough);
+    });
+    await visit('/vault/clients/history');
+    assert.equal(currentURL(), '/vault/clients/history', 'clients/history URL is correct');
+    await click('[data-test-popup-menu-trigger="month"]');
+    const dropdownListMonths = this.element.querySelectorAll('[data-test-month-list] button');
+    // TODO CMB change how selected b/c depending on time of year first month or year may be disabled
```

Comment:
Could you look for the first button that isn't disabled?

Reply:
That's what I was thinking!